docs/what.rst

```diff
@@ -0,0 +1,23 @@
+=====================
+What is a Certificate?
+=====================
+
+An SSL *certificate* contains a public key and a private key that work together to enable
+a web browser to communicate with your server securely over an encrypted SSL (secure socket layer)
+connection. The certificate also identifies your website so that the browser (and the person behind it)
+can trust your identity. SSL certificates have mainly been used to provide secure,
+encrypted communication for sensitive information like financial and health data, but
+we now recommend all websites run on SSL to prevent eavesdropping and content hijacking.
```

Comment:
In relation to this suggestion and a few other suggestions above that you made last night, - I am quoting from my comment above:

> Yet another suggestion: While a big focus of certbot (and LetsEncrypt) is certificates for websites, these certificates are also used for other servers (IMAP/POP3, MTA/MSA, ... ). I think the documentation should be talking more generally about "servers" (as opposed to webservers).